src/main/java/br/com/harbitech/school/activity/Activity.java

```diff
@@ -24,42 +26,16 @@
     @ManyToOne(fetch = FetchType.LAZY)
     private Section section;
```

Comment:
Será que vale um `@NotNull` aqui?

```suggestion
    @NotNull
    private Section section;
```